library/CoreData/Base/iCloudCDStack.swift

```diff
@@ -333,7 +335,7 @@ public class iCloudCDStack: DefaultCDStack
     
     :param: notification Notification with these changes
     */
-    internal func persistentStoreDidImportUbiquitousContentChanges(notification: NSNotification)
+    @objc internal func persistentStoreDidImportUbiquitousContentChanges(notification: NSNotification)
```

Comment:
Why @objc ? Does it have conflict with something if we do not specify the Objective-C compatibility? @dchavezlive